pkg/cmd/pr/checks/checks_test.go

```diff
@@ -69,6 +69,7 @@ func Test_checksRun(t *testing.T) {
 		wantOut string
 		nontty  bool
 		wantErr bool
+		throw   bool
```

Comment:
I think we could rename `wantErr` to `nonSuccessfulChecks` and `throw` to `wantErr` to make the purpose of them more clear.

Reply:
that make sense